lenses/chrony.aug

```diff
@@ -254,6 +263,13 @@ module Chrony =
                       . ( space . [ label "port" . store integer ] )?
                       . eol ]
 
+    (* View: bcast
+         confdir and sourcedir have specific syntax
+    *)
+    let dir_list = [ Util.indent . key /(conf|source)dir/
+                      . ( space . [ key no_space ] )+
```

Comment:
This is not the most common way of storing a list of values in Augeas (which would be to use either multiple nodes with the same label, or a `seq`). Is there a reason for doing it this way?

Reply:
No good reason. I just didn't know the conventions. It should be fixed now. Thanks.